### PR TITLE
tests: add test for issue 4751

### PR DESCRIPTION
**Pull request by hsadia538**
Previous PR #969 

Redmine [ticket](https://redmine.openinfosecfoundation.org/issues/4751)

Describe changes:
- Update commit message
- Update Readme file
- Update `test.yaml` to add `min-version`

### CODE REVIEW

File: tests/dns-eve-log-https-only/test.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+requires:
+  min-version: 7
+checks:
```
Comment (inashivb) [inline tests/dns-eve-log-https-only/test.yaml:3]: just a minor nit: for legibility, we should probably have a newline here.

**Review by inashivb — CHANGES_REQUESTED** [review]
Nice job. :)
Minor: the commit message says "Tickat"

**Comment by hsadia538** [comment]
Followed by #975

File: tests/dns-eve-log-https-only/test.yaml (hunk 1 above)
Comment (hsadia538) [reply tests/dns-eve-log-https-only/test.yaml:3]: I added a line after requires block(new PR). But now I am confused. Were you referring to line 3?

**Pull request closed without merging**